### PR TITLE
Fix test name duplicate for hugepages tests

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

To avoid getting errors from junit-merger we make the test name for 2Gi
with guest memory configuration unique.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @rmohr 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/vmi_configuration_test.go [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			},
 				table.Entry("[Serial][test_id:1671]hugepages-2Mi", "2Mi", "64Mi", "None"),
 				table.Entry("[Serial][test_id:1672]hugepages-1Gi", "1Gi", "1Gi", "None"),
-				table.Entry("[Serial][test_id:1672]hugepages-1Gi", "2Mi", "70Mi", "64Mi"),
+				table.Entry("[Serial][test_id:1672]hugepages-2Mi-guest-memory", "2Mi", "70Mi", "64Mi"),
```
Comment (rmohr) [inline tests/vmi_configuration_test.go:1116]: can we make the `-guest-memory` prefix a little bit more descriptive? Something like with guest memory explicitly set?

**Review by rmohr — CHANGES_REQUESTED** [review]

File: tests/vmi_configuration_test.go (hunk 1 above)
Comment (dhiller) [reply tests/vmi_configuration_test.go]: Done

**Comment by rmohr** [comment]
/retest

introduced a small mistake in the migrations job definition when making them mandatory. Should be fixed now.

**Pull request merged**